### ISSUE TITLE
Vanced Manager can't install YT Vanced/YT Music Vanced

### ISSUE THREAD

**Issue by DucCuongTaSE** [YTVanced/VancedManager#604]
**Please only open the issue if the following is true**
- This is an issue in the Vanced Manager and ONLY Vanced Manager (NOT Youtube Vanced/Music/microG)

**Phone Specifications:**
- Brand: Nokia 8
- Operating System: Android (CrDroid)
- Android Version: 11
- Vanced Manager Version: 2.6.0

![Screenshot_20210630-143208380 1](https://user-images.githubusercontent.com/86711042/123923686-6ada5900-d9b3-11eb-8331-a26f08b7f77c.jpg)

**Comment by X1nto** [comment]
Are you using magisk?

**Comment by DucCuongTaSE** [comment]
> Are you using magisk?

yes

**Comment by ghost** [comment]
Install the `BusyBox for Android NDK` Magisk Module

**Comment by X1nto** [comment]
> Install the `BusyBox for Android NDK` Magisk Module

Hmm, I don't recall manager needing BusyBox to work properly, is that really a thing?

**Comment by ghost** [comment]
> > Install the `BusyBox for Android NDK` Magisk Module
> 
> Hmm, I don't recall manager needing BusyBox to work properly, is that really a thing?

Yeah, I assume it's because some ROMs (especially older ones) have a bad Unix tools implementation. BusyBox seems to remedy most of the time when this comes up.

Not in this case tho it seems, we're troubleshooting rn in the DC.

**Comment by DucCuongTaSE** [comment]
idk, my rom is 26 May 2021

**Comment by X1nto** [comment]
> > > Install the `BusyBox for Android NDK` Magisk Module
> > 
> > 
> > Hmm, I don't recall manager needing BusyBox to work properly, is that really a thing?
> 
> Yeah, I assume it's because some ROMs (especially older ones) have a bad Unix tools implementation. BusyBox seems to remedy most of the time when this comes up.
> 
> Not in this case tho it seems, we're troubleshooting rn in the DC.

That's interesting, perhaps we could use [BusyBox addon](https://github.com/topjohnwu/libsu#busybox) for [libsu](https://github.com/topjohnwu/libsu)

**Comment by ghost** [comment]
> > > > Install the `BusyBox for Android NDK` Magisk Module
> > > 
> > > 
> > > Hmm, I don't recall manager needing BusyBox to work properly, is that really a thing?
> > 
> > 
> > Yeah, I assume it's because some ROMs (especially older ones) have a bad Unix tools implementation. BusyBox seems to remedy most of the time when this comes up.
> > Not in this case tho it seems, we're troubleshooting rn in the DC.
> 
> That's interesting, perhaps we could use [BusyBox addon](https://github.com/topjohnwu/libsu#busybox) for [libsu](https://github.com/topjohnwu/libsu)

Looks like a good idea to at least test out.

**Comment by ch1nh0w** [comment]
> Install the `BusyBox for Android NDK` Magisk Module

this brilliant suggestion solved my download error on Lineage OS with Magisk, thank you very much! :)
